dashboard/config/routes.rb

```diff
@@ -856,10 +856,6 @@
 
     get '/dashboardapi/v1/regional_partners/find', to: 'api/v1/regional_partners#find'
     get '/dashboardapi/v1/regional_partners/show/:partner_id', to: 'api/v1/regional_partners#show'
-    get '/dashboardapi/v1/pd/application/applications_closed', to: 'pd/professional_learning#applications_closed'
-    get '/dashboardapi/v1/pd/workshops_as_facilitator_for_pl_page', to: 'pd/professional_learning#workshops_as_facilitator_for_pl_page'
-    get '/dashboardapi/v1/pd/workshops_as_organizer_for_pl_page', to: 'pd/professional_learning#workshops_as_organizer_for_pl_page'
-    get '/dashboardapi/v1/pd/workshops_as_program_manager_for_pl_page', to: 'pd/professional_learning#workshops_as_program_manager_for_pl_page'
```

Comment:
I'm a little bit wondering if we leave these here and write a comment that they're deprecated (and eventually delete them). I'm thinking of the case of:
- user opens the My PL page
- the DTP with this change completes
- user switches tabs and the request to fetch workshops 404s because these routes no longer exists

It's definitely an edge case, though, so open to feedback!

Reply:
Sounds good, yeah that totally makes sense! I'll add them back in